Sources/StreamChatUI/ChatMessageList/ChatMessageListTitleView.swift

```diff
@@ -5,44 +5,51 @@
 import StreamChat
 import UIKit
 
-// TODO: Audit
-public class ChatMessageListTitleView<ExtraData: ExtraDataTypes>: UIView, UIConfigProvider {
-    public var title: String? {
-        get { titleLabel.text }
-        set { titleLabel.text = newValue }
+/// A view that is used as a wrapper for status data in navigationItem's titleView
+open class _ChatMessageListTitleView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
+    /// Content of the view that contains title (first line) and subtitle (second nil)
+    open var content: (title: String?, subtitle: String?) = (nil, nil) {
+        didSet {
+            updateContent()
+        }
     }
     
-    public var subtitle: String? {
-        get { subtitleLabel.text }
-        set { subtitleLabel.text = newValue }
-    }
+    /// Label that represents the first line of the final titleView
+    open private(set) var titleLabel: UILabel = UILabel()
+    
+    /// Label that represents the second line of the final titleView
+    open private(set) var subtitleLabel: UILabel = UILabel()
     
-    private weak var titleLabel: UILabel!
-    private weak var subtitleLabel: UILabel!
+    /// A view that joins the `titleLabel` and `subtitleLabel` in vertical stack
```

Comment:
nit: It's usually better to write the documentation comments such that they are valid even if something changes. i.e. what if someone decides to replace `subtitleLabel` with something else and change the axis to `.horizontal`? :)

IMO the most important thing about this view is that it acts as the main container for the subvies